components/Common/CrossLink/index.stories.tsx

```diff
@@ -0,0 +1,55 @@
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+import CrossLink from './index';
+
+type Story = StoryObj<typeof CrossLink>;
+type Meta = MetaObj<typeof CrossLink>;
+
+export const Prev: Story = {
+  render: () => (
+    <div className="w-[305px] bg-white p-2">
+      <CrossLink
+        direction="left"
+        text="How to install Node.js"
+        url="https://nodejs.dev/en/learn/how-to-install-nodejs/"
+      />
+    </div>
+  ),
+};
+
+export const Next: Story = {
+  render: () => (
+    <div className="w-[305px] bg-white p-2">
+      <CrossLink
+        direction="right"
+        text="How much JavaScript do you need to know to use Node.js?"
+        url="https://nodejs.dev/en/learn/how-much-javascript-do-you-need-to-know-to-use-nodejs/"
+      />
+    </div>
+  ),
+};
+
+export const PrevDark: Story = {
```

Comment:
Once the other PR gets merged, we should only have stories for variants and not for dark/white